### PR TITLE
Create model card for pre-trained NLI models.

### PR DESCRIPTION
**Pull request by easonnie**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dimiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests), 
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to the it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests? 

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

File: model_cards/ynie/roberta-large-snli_mnli_fever_anli_R1_R2_R3-nli/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+This is a strong pre-trained RoBERTa-Large NLI model.
```
Comment (julien-c) [inline model_cards/ynie/roberta-large-snli_mnli_fever_anli_R1_R2_R3-nli/README.md:1]:
```suggestion
---
datasets:
- snli
- anli
license: ___
---
This is a strong pre-trained RoBERTa-Large NLI model.  
```

**Review by julien-c — CHANGES_REQUESTED** [review]
This is great, thanks for sharing!

Could you add metadata links to the datasets that were used, using `huggingface/datasets` identifiers (listed in https://huggingface.co/datasets)? This will help tremendously for discoverability.

If a dataset isn't in `datasets` yet, you can still add it in the metadat (this will prompt us – or a community member – to add it)

cc @lhoestq and @thomwolf on the `datasets` side, @mapmeld on the metadata side, and @joeddav for his interest on NLI models

**Comment by easonnie** [comment]
Hi @julien-c, thank you so much for the pointer. I have added the dataset identifier and license.

**Comment by julien-c** [comment]
Thanks! Your model is now linked from https://huggingface.co/datasets/anli

**Pull request merged**